draft-irtf-cfrg-bbs-signatures.md

```diff
@@ -202,8 +199,8 @@ e
 r
 : The prime order of the G1 and G2 subgroups.
 
-P1, P2
-: points on G1 and G2 respectively. For a pairing-friendly curve, this document denotes operations in E1 and E2 in additive notation, i.e., P + Q denotes point addition and x \* P denotes scalar multiplication. Operations in GT are written in multiplicative notation, i.e., a \* b is field multiplication.
+BP, BP'
```

Comment:
Why not BP1 and BP2, the `'` character might be hard to understand in the middle of the spec.

Reply:
The indent was to use the notation from[ pairing-friendly curves](https://www.ietf.org/archive/id/draft-irtf-cfrg-pairing-friendly-curves-08.html#name-bls-curves-for-the-128-bit-) draft. But yea its hard to see. Changed to use `BP1` and `BP2`.